[ZRH-2023] Add and remove organizers

romanoroth opened the pull request:
Email with Email-Addresses will follow soon.....

MikeRosTX reviewed (changes requested):
So, from this [PR #11696](https://github.com/devopsdays/devopsdays-web/pull/11696) you're requesting we remove Carsten Plum from team email dl and slack, then add Alina Ehrle team email dl and slack. This is completed, done.

But we cannot merge the PR until the following are resolved:

1. Tobias Brunner is listed on team email dl and slack, but not on the Contact.md page.
2. Franziska Bühler is listed in slack and on Contact.md page, but not on team email dl.

Please tell us how you would like to proceed.

romanoroth commented:
Hi @MikeRosTX 
Thank you. 
I've removed Vitorio from the team list and removed Tobias Brunner from the email and added Franziska Bühlers Email.
Sorry for that. 
Now the email and the team list should be in sync.
